core/src/test/java/hudson/FilePathTest.java

```diff
@@ -325,6 +328,33 @@ private FilePath createFilePath(final File base, final String... path) throws IO
         FileUtils.touch(building);
         return new FilePath(building);
     }
+    
+    
+    /**
+     * Checks that big files (>8GB) can be archived and then unpacked.
+     * This test is disabled by default due the impact on RAM.
+     * The actual file size limit is 8589934591 bytes.
+     * @throws Exception test failure
+     */
+    @Issue("JENKINS-10629")
+    @Ignore
+    @Test public void archiveBigFile() throws Exception {
+        assumeTrue(!Functions.isWindows());
```

Comment:
Why - this will run on Windows as well as well.  It is only run when a user explicitly wants to run it.
It's not like the Linux machine is checked for sparse file support first.